agentai/api.py

```diff
@@ -1,31 +1,36 @@
 """
 API functions for the agentai package
 """
-from typing import Callable
+from typing import Callable, List
```

Comment:
prefer `list`, `dict` over `List`, `Dict` in Python>=3.9

Reply:
Done.